config/config.go

```diff
@@ -1029,6 +1036,14 @@ func SetupViper(v *viper.Viper, filename string, bidderInfos BidderInfos) {
 	v.SetDefault("debug.timeout_notification.fail_only", false)
 	v.SetDefault("debug.override_token", "")
 
+	v.SetDefault("tmax_adjustments.enabled", false)
+	v.SetDefault("tmax_adjustments.auction_max", 900)
+	v.SetDefault("tmax_adjustments.video_max", 900)
+	v.SetDefault("tmax_adjustments.amp_max", 900)
```

Comment:
The default values for these should be `0` which would equivalent to these values not set at all and the behavior being same as it is today. This is because a host who hasn't started using these values shouldn't automatically see a change in behavior. They should have to specifically set these values to allow PBS to modify tmax for bidder requests

Reply:
Thanks for the explanation. https://github.com/prebid/prebid-server/pull/2723/commits/4604aa4b898b8e3f660b192bb0c779ec37a6d2f6 makes changes to set zero as default value